build.gradle

```diff
@@ -124,9 +124,9 @@ ext {
   composeUiToolingPreview = "androidx.compose.ui:ui-tooling-preview:$compose_version"
   composeTestJUnit = "androidx.compose.ui:ui-test-junit4:$compose_version"
   composeUiTooling = "androidx.compose.ui:ui-tooling:$compose_version"
-  composeActivity = "androidx.activity:activity-compose:1.3.1"
+  composeActivity = "androidx.activity:activity-compose:1.3.0"
```

Comment:
for some reason this needs to be 1.3.0 - it seems there's a strict dependency onto this from compose. upgrading it will result in a version conflict